config/config.go

```diff
@@ -161,6 +161,7 @@ type HeartbeatConfig struct {
 	MinTimeToWaitBetweenBroadcastsInSec int
 	MaxTimeToWaitBetweenBroadcastsInSec int
 	DurationInSecToConsiderUnresponsive int
+	HbmiRefreshInterval                 uint32
```

Comment:
HbmiRefreshIntervalInSec* as to be more specific

Reply:
fixed

Reply:
fixed